packages/fast-components-styles-msft/src/utilities/color/accent-fill.ts

```diff
@@ -21,7 +22,7 @@ import { getSwatch, isDarkMode, Palette, palette, PaletteType } from "./palette"
 function accentFillAlgorithm(contrast: number): DesignSystemResolver<FillSwatchFamily> {
     return (designSystem: DesignSystem): FillSwatchFamily => {
         const accentPalette: Palette = palette(PaletteType.accent)(designSystem);
-        const accent: Swatch = accentSwatch(designSystem);
+        const accent: Swatch = accentBaseColor(designSystem);
```

Comment:
Am I missing where this is a function that accepts a designSystem value? The only reference I see from `../design-system` is:

```
    /**
     * The accent color, which the accentPalette is based on
     */
    accentBaseColor: string;
```

Reply:
It's a helper function all the way down in `packages/fast-components-styles-msft/src/utilities/design-system.ts`

Reply:
thx